frontend/src/pages/modelServing/screens/projects/kServeModal/ManageKServeModal.tsx

```diff
@@ -224,14 +224,14 @@ const ManageKServeModal: React.FC<ManageKServeModalProps> = ({
 
   return (
     <Modal
-      title="Deploy model"
+      title={!editInfo ? 'Deploy model' : 'Edit model'}
```

Comment:
The failing tests seem to be that the tests are looking for "Deploy model" but seeing "Edit model". These tests are in `servingRuntimeList.cy.ts`:

- Successfully submit KServe Modal on edit
- Check model size rendered with ServingRuntime size and no InferenceServiceSize
- Check model size rendered with InferenceService size
- Check model size rendered with InferenceService custom size

they all seem to be opening on an existing model (so it would be editing), so the tests should be updated to use the edit version